### DISCUSSION TITLE
generate header file

### DISCUSSION THREAD

**Discussion by kamil-zubair** [hsutter/cppfront#783] [Q&A] [answered]
is there a way to generate header file ? I plan to rewrite the utility functions in my personal project with cpp2 and I need the header file so I can include it in my other cpp1 files

**Comment by JohelEGP** [comment, marked as answer]
See #594.
There, commit 347c1c26f5651d7706001c18e2bf139f4e74b5fa is mentioned, which added support for `.h2` header files.
